lighthouse-core/runner.js

```diff
@@ -33,6 +34,8 @@ class Runner {
 
     const config = opts.config;
 
+    Sentry.init(opts.flags.disableErrorReporting === false, opts.environmentData || {});
```

Comment:
definitely should be `opts.flags.enableErrorReporting === true`

Reply:
ok right